lib/facility_factory.rb

```diff
@@ -0,0 +1,36 @@
+require './lib/facility'
+
+class FacilityFactory
```

Comment:
Though you didn't get to the third dataset, you've shown enough that I am confident that you could add additional datasets, so I'm going to consider this as iteration3 finished.